lib/nvme/nvme_ctrlr_cmd.c

```diff
@@ -408,7 +421,10 @@ nvme_ctrlr_cmd_abort(struct spdk_nvme_ctrlr *ctrlr, uint16_t cid,
 	cmd->opc = SPDK_NVME_OPC_ABORT;
 	cmd->cdw10 = (cid << 16) | sqid;
 
-	return nvme_ctrlr_submit_admin_request(ctrlr, req);
+	rc = nvme_ctrlr_submit_admin_request(ctrlr, req);
+	ctrlr->curr_abort_count++;
```

Comment:
curr_abort_count should probably only be incremented if the nvme_ctrlr_submit_admin_request() call was successful.